modules/darwin/common/telegraf.nix

```diff
@@ -7,6 +7,7 @@
         prometheus.metric_version = 2;
         system = { };
         mem = { };
+        swap = { };
         disk.tagdrop = {
           fstype = [ "tmpfs" "ramfs" "devtmpfs" "devfs" "iso9660" "overlay" "aufs" "squashfs" ];
```

Comment:
Pretty sure most of them don't exist on macos i.e. overlay aufs squashfs

Reply:
Yes, I just copied `disk.tagdrop` from the srvos module so it's easier to keep them in sync.